### DISCUSSION TITLE
Affichage de documents externes via Papi Web

### DISCUSSION THREAD

**Discussion by wilchel** [pascalaubry/papi-web#32] [Ideas]
Bonjour,
Pensez-vous qu il serait possible d'afficher via PapiWeb des documents externes sur les postes clients , par exemple
Avec Papi , on peut exporter le classement général en html, s'il était possible d'afficher cette page html sur les postes clients, cela permettrait aux joueurs de connaitre leur position entre les rondes
La fonction intéressante serait d'avoir un répertoire ou papiweb effectuerait une scrutation de fichier et afficherait dans une page (ou onlget) la liste des fichiers disponibles ( règlement, affiche tournoi, ....)
Merci pour ce super outil :)

**Comment by Amaras** [comment]
Bonjour, je réponds après un peu de temps de réflexion.

L'affichage d'un document statique n'est pas dans nos priorités, mais envisageable.
Je garde l'idée en tête, cependant cela peut amener un risque de sécurité si l'implémentation est bâclée, je reste donc prudent.

**Comment by timothyarmes** [comment]
> par exemple Avec Papi , on peut exporter le classement général en html, s'il était possible d'afficher cette page html sur les postes clients

Pour info, PapiWeb affiche désormais le classement général.

**Comment by pascalaubry** [comment]
Bonjour Bachir,
Depuis ta question originelle, les écrans de type « Image » ont également été ajoutés pour les clients, à inclure par exemple dans les écrans rotatifs.
